css/index.css

```diff
@@ -1 +1,306 @@
\ No newline at end of file
+/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
+
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+	font-size: 62.5%; 
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* -------------------------------------- */
+
+.navigation {
+
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	max-width: 75%;
+	
+}
+
+a {
+	margin: 3%;
+	font-weight: bold;
+	font-size: 1.8rem;
+	padding-top: 24px;
+	padding-bottom: 16px;
+	padding-right: 32px;
+	color: #E2E5E8;
+	text-decoration: none;
+	max-width: 75%;
+}
+
+
+.idea-img {
+	display: inline-block;
+	position: absolute;
+	top: 45px;
+	right: 175px;
+	
+}
+
+.header {
+    display: flex;
+    justify-content: center;
+    align-content: space-around;
+    margin: 2rem;
+}
+
+.header h1 {
+	font-size: 10rem;
+	text-align: center;
+}
+
+.getstarted-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: flex-start;
+	padding-right: 500px;
+	font-weight: bold;
+	font-size: 1.5rem;
+	
+}
+
+.top-section-header {
+    display: flex;
+    justify-content: space-around;
+    align-content: column;
+    margin: 4rem;
+    padding-top: 45px;
+	border-top: 2px solid black;
+	font-size: 2rem;
+}
+
+.top-section {
+    display: flex;
+    justify-content: center;
+    align-content: column;
+    /* margin: 24px; */
+	margin-right: 12px;
+	font-size: 1.8rem;
+	line-height: 1.25;
+}
+
+.middle-img {
+    display: flex;
+    justify-content: space-around;
+    align-content: column;
+	margin: 0 auto;
+	padding: 2% 0%;
```

Comment:
Try to avoid using percentages for vertical (top & bottom) padding or margins because screen size is generally dictated by its width